binr/r2r/r2r.c

```diff
@@ -387,6 +397,19 @@ int main(int argc, char **argv) {
 	}
 	r_pvector_clear (&workers);
 
+	ut64 seconds = (r_sys_now () - time_start) / 1000000;
```

Comment:
use the constants!

Reply:
you wrote that code in the first place :-P

Reply:
I think we should do that change in another pr, where we move some stuff to r_util.